pom.xml

```diff
@@ -131,6 +131,11 @@
       <version>2.4.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
What is this used for?

Reply:
It was used for a list-ordered map, but after incorporating your changes we don't need it. I'll remove it. I removed it from CIRC-265 but didn't do it here. Good catch